test/jasmine/tests/register_test.js

```diff
@@ -1,6 +1,27 @@
 var Plotly = require('@lib/index');
 
 describe('the register function', function() {
+    'use strict';
+
+    var Plots = Plotly.Plots;
+
+    beforeEach(function() {
+        this.modulesKeys = Object.keys(Plots.modules);
```

Comment:
Actually on second look, I'm ok with that. Didn't know that it was jasmine convention and that the `this` is cleared between specs.